src/misc.ts

```diff
@@ -125,6 +126,25 @@ export const transformPaymentStatus = (
   };
 };
 
+export const transformChargedStatus = (status: OrderChargeStatusEnum, intl: IntlShape) => {
+  switch (status) {
+    case OrderChargeStatusEnum.OVERCHARGED:
+      return {
+        localized: intl.formatMessage({
```

Comment:
question: Why not `label`?

Reply:
I was following a pattern for payment status, it uses `localized` (`transformOrderStatus` fn)